src/Categories.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import Categories from './Categories';
+
+import categories from '../fixtures/categories';
+
+describe('Categories', () => {
+  const handleClick = jest.fn();
+
+  const renderCategories = () => render((
+    <Categories 
+      categories={categories} 
+      onClickCheck={handleClick} 
+    />
+  ))
+
+  it('renders categories', () => {
+    const { queryByText } = renderCategories();
+
+    categories.forEach((category) => {
+      expect(queryByText(category.name)).not.toBeNull();
+    })
+  })
+
+  it('clicks category buttons', () => {
+    const { queryByText } = renderCategories();
+
+    categories.forEach((category) => {
+      fireEvent.click(queryByText(category.name));
+    })  
+
+    expect(handleClick).toBeCalledTimes(categories.length);
```

Comment:
각 버튼을 누를 때 마다 핸들러 함수가 호출되도록 하는게 더 의미가 잘 드러날 것 같아요.
```suggestion
    categories.forEach((category) => {
      fireEvent.click(queryByText(category.name));
      
      expect(handleClick).toBeCalled();
    })  
```